packages/web/src/components/host-remix-contest-modal/HostRemixContestModal.tsx

```diff
@@ -168,7 +210,44 @@ export const HostRemixContestModal = () => {
           <Text variant='body'>{messages.description}</Text>
           <Flex direction='column' gap='l'>
             <Text variant='title' size='l'>
-              {messages.deadlineTitle}
+              {messages.descriptionLabel}
+            </Text>
+            <TextAreaV2
+              aria-label='contestDescription'
```

Comment:
we should connect the title text to the text area using <label> + htmlFor + id for description and prizes. We should ideally do this for submission deadline as well.